openhcl/openhcl_boot/src/main.rs

```diff
@@ -543,7 +558,12 @@ fn shim_main(shim_params_raw_offset: isize) -> ! {
         p.isolation_type == IsolationType::None || static_options.confidential_debug;
 
     let boot_reftime = if p.isolation_type.is_hardware_isolated() {
-        None
+        let result = get_hw_tsc_time(p.isolation_type);
```

Comment:
See the other comment about generalizing this `reference_time` function. I wonder if we should instead also move the implementation to `minimal_rt`?

Reply:
Agreed. It will be useful for future changes that need tdcall-based hypercalls needed in openhcl_boot.